cmake/Modules/MacroQbtCommonConfig.cmake

```diff
@@ -70,8 +76,12 @@ macro(qbt_common_config)
     endif()
 
     if (MSVC)
-        target_compile_options(qbt_common_cfg INTERFACE /guard:cf)
-        target_link_options(qbt_common_cfg INTERFACE /guard:cf
+        target_compile_options(qbt_common_cfg INTERFACE
+            /guard:cf
+            /wd4819
```

Comment:
Is it the thing we talked about in https://github.com/qbittorrent/qBittorrent/pull/14156#discussion_r552025968?

Reply:
Yes.